### ISSUE TITLE
Break modal should trap keyboard focus

### ISSUE THREAD

**Issue by chrisauclair** [cs50/video50#87]
When the break modal menu takes control of the player between chapters, keyboard focus should be trapped in the modal. If a keyboard user tries to tab past the last button in the modal, they should go back to the first one. Likewise if they use `shift + tab` for navigation. When the modal exits, focus should be restored to where it was before the modal opened.

**Comment by lukejacksonn** [comment]
This is fixed in https://github.com/cs50/video50/pull/123
